packages/react-look/modules/plugins/friendlyClassName.js

```diff
@@ -17,7 +17,13 @@ export default function friendlyClassNames({ StyleContainer, Component, element,
       if (classMapping.has(cls)) {
         className += className + className !== '' ? ' ' : '' + classMapping.get(cls)
       } else {
-        const newClass = classNameTemplate(cls, Component, element)
+        let newClass
+        // checking if config is undefined to avoid breaking existing test
```

Comment:
Hmm...wasn't sure whether this would be an interface that could be used by people using react-look or just used internally. 

If the latter, it's okay to add a `config: {}` to the first test. 
If the former, wouldn't it still be possible for user to not pass the config param (in which case this test reflects that use case)?